routes/article.js

```diff
@@ -13,24 +13,24 @@ router.post('/vote', (req, res) => {
     if (amount === 0) {
         res.status(500).send({
             err: "Invalid value for vote",
-            reqBody: req.body
+            givens: req.body
         })
         return
     }
 
     let articleChanged = null
     Article.findById(req.body.id)
-        .then((article) => {
+        .then(article => {
```

Comment:
You have the rule for braces messed up, unless it's a oneline express arrow function, you don't omit the braces. Checkout airbnb's style guide, we're trying to follow it (although we did throw out the semi-colons) 
https://github.com/airbnb/javascript#arrow-functions